CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCImportController.java

```diff
@@ -193,7 +193,20 @@ protected void updateSampleData() {
 
         if (configurationPane != null) {
             configurationPane.setSampleData(currentColumns, currentData);
+            openConfigPane(!currentColumns[0].isBlank());
+            enableButton(currentColumns[0].isBlank());
```

Comment:
Looks like further up the function there is the possibility of `currentColumns` being length 0, in which case this will throw an `ArrayOutOfBoundsException` so you'll probably want to add in a boolean condition for that. I would also switch to using `StringUtils.isBlank()` and `StringUtils.isNotBlank()` is case the value of `currentColumns[0]` is null